etc/oods/config/ncsa/at_oods.yaml

```diff
@@ -36,7 +36,8 @@ cacheCleaner:
         - /data/lsstdata/NTS/auxTel/oods/butler/repo/raw
         - /data/lsstdata/NTS/auxTel/oods/butler/badfiles
         - /data/lsstdata/NTS/auxTel/oods/butler/staging
-        - /data/lsstdata/NTS/auxTel/oods/gen3butler/repo/raw
+        - /data/lsstdata/NTS/auxTel/oods/gen3butler/raw
+        - /data/lsstdata/NTS/auxTel/oods/gen3butler/repo/LATISS/raw
```

Comment:
I think this should just be `/data/lsstdata/NTS/auxTel/oods/gen3butler/repo/LATISS`; any user-created datasets are fair game for the OODS to clean up.

Reply:
That points at the old directory where the butler stored the moved files.  I believe any new user datasets would not go under that directory.  The whole thing will end up going away (and this entry removed) in about two weeks when the files are completely removed.

Reply:
I see, but then should there be an entry for the new `/data/lsstdata/repo/LATISS/u`?

Reply:
I'd considered that, but was unsure of the intent of that directory and it's usage by the OODS.  I had thought that the butler itself would be RW (which I just realized we'd need to change, since gen3.sqlite3 has permissions 744, and it should be at least 774, if not 777, so users could change things in it).   

You'll note however, that if "move" were used by a butler user (not the OODS),  first order files under /repo/LATISS/u could be deleted by OODS as saluser, but not user created directories below that. If there is any kind of directory structure under /repo/LATISS/u set by the user using "move", the files below it can't be deleted by the OODS.  

In other words, user1 creating the file /repo/LATISS/u/file1 could be deleted by saluser, since /repo/LATISS/u is 777;  but user1 creating the file /repo/LATISS/u/dir1/file1 could not be deleted by saluser, since "file1" is owned by user1 and so it its parent directory "dir1".    I don't think butler allows the user has direct control over how the files get named using "move", since in previous OODS ingests, the butler creates an extended directory structure.  I think it needs to do that, because otherwise there could be many files under /repo/LATISS/u would which slowly degrade performance.

One possible solution is to have the butler always write files with the correct permissions to allow that.  I don't believe it currently does.  I think it defaults to creating files and directories using the users's umask. We'd need the butler to enforce permissions that would allow "saluser" to delete files or come up with another solution.